src/python.rs

```diff
@@ -29,14 +29,22 @@ fn convert_config(py: Python, v: ConfigVal) -> PyObject {
 
 // Transation between rust extraction type and Python dictionary
 fn pythonize(py: Python, extraction: Extraction) -> PyResult<PyObject> {
-    let refs: Vec<Vec<String>> = extraction
-        .refs
-        .into_iter()
-        .map(|x| match x {
-            (a, Some(b)) => vec![a, b],
-            (a, None) => vec![a],
-        })
-        .collect();
+    let refs = PyList::empty(py);
+
+    for r in extraction.refs.iter() {
+        let pyref = PyList::empty(py);
```

Comment:
I think it might be clearer to make each ref a `PyDict` with named keys rather than a `PyList`- but this does work. You just have to remember the order on the dbt-core implementation side as well.